azure-pipelines.yml

```diff
@@ -538,3 +538,58 @@ jobs:
     inputs:
       targetPath: '$(Build.ArtifactStagingDirectory)'
     condition: succeededOrFailed()
+
+- job: 'Fuzzing'
+  timeoutInMinutes: 60
+  dependsOn: 'GetReleaseTag'
+  condition: always()

```

Comment:
I feel strongly that this condition should be changed so that the `fuzzing` job only runs for the master branch. I don't think we should overload the onefuzz task by submitting artifacts for every PR build as the task doesn't immediately give us any information about fuzzing bugs. Those are autogenerated by the onefuzz service and sent as an ADO notification. It would be less noisy if it only ran for a master branch commit because that is actual code that is checked in and not being worked on. 

If there are no objections, I will change this after this PR is reviewed and approved so that we can see it run to completion.

Reply:
Yes, if "the task doesn't immediately give us any information about fuzzing bugs", then we should consider to run it only on ci builds